xgboost/src/main/java/hivemall/xgboost/XGBoostUDTF.java

```diff
@@ -269,44 +270,35 @@ public void checkTargetValue(double target) throws HiveException {}
     public void process(Object[] args) throws HiveException {
```

Comment:
unnecessary double empty lines
https://github.com/maropu/incubator-hivemall/blob/faa5433e0e6253a19e26c7cba9aebc8b3c87f18e/xgboost/src/main/java/hivemall/xgboost/XGBoostUDTF.java#L268

Are you applied formatter?